### PR TITLE
Review 7/4/22

### PR DESCRIPTION
**Pull request by rockerinnight**

### CODE REVIEW

**Comment by DienMinh** [comment]
@rockerinnight

File: src/app/button/button.component.scss [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-  border-radius: 50%;
   cursor: pointer;
-}
-// can use class:
-/*
-  d-block for line 1
-  rounded-circle for line 9
-*/
\ No newline at end of file
+}
```
Comment (rockerinnight) [inline src/app/button/button.component.scss:9]: end of line

File: src/app/Pipes/text-overflow.pipe.ts [hunk 2]
```diff
@@ -5,11 +5,10 @@ import { Pipe, PipeTransform } from '@angular/core';
 })
 export class TextOverflowPipe implements PipeTransform {
   transform(value: string, limit: number, ...args: unknown[]): unknown {
```
Comment (rockerinnight) [inline src/app/Pipes/text-overflow.pipe.ts:7]: implement like this: transform(value: string, limit?: number, ...) { }
if do this, when using this pipe wont need to input the limitation, default limit will be 30 ( {{ ... | textOverFlow }} )

**Comment by rockerinnight** [comment]
@DienMinh

**Pull request merged**